docs/howto/write-unit-tests-for-a-charm.md

```diff
@@ -162,6 +162,50 @@ def test_charm_runs(my_charm):
 If you use pytest, you should put the `my_charm` fixture in a top level `conftest.py`, as it will likely be shared between all your unit tests.
 ```
 
+## Reusing state
+
+Each test is typically an isolated test of a single event handler. If you need to use the same input state for several events, we recommend using a fixture that produces the input state. For example:
```

Comment:
```suggestion
Each test is typically an isolated test of a single state transition. If you need to use the same input state for several events, we recommend using a fixture that produces the input state. For example:
```

"event handler" is a thing in Ops, and a charm can and does have multiple attached to a single Juju event or Ops lifecycle event.